docs/privacy-policy.md

```diff
@@ -0,0 +1,127 @@
+Privacy Policy
+==============
+
+**DRAFT**
+
+Your personal privacy is an important factor that the Sempervirens project and Numfocus 
+(our legal entity) considers in the development of each of our products and services. 
+We are committed to gathering NO personal information through sempervirens. We are 
+committed to being transparent and open. This Privacy Policy explains generically how 
+we receive information from you, and what we do with that  information once we have it.
+
+What do we mean by "personal information?"
+------------------------------------------
+
+The phrase "personal information" means information which identifies you, such as your 
+name, email address, twitter account names, or other user names. Any information that 
+falls outside of this is "non-personal information."
+
+Combining non-personal information with personal information we consider to be 
+"personal information."  Such combinations will not be collected. Removing all 
+personal information creates a data set that is deemed "non-personal information" 
+and may be collected.
+
+How do we learn information about you?
+---------------------------------------
+
+We learn non-personal information about you when:
+
+* you give it to us directly (e.g., when you choose to send us crash reports);
+* we collect it automatically through our products and services (e.g., the Sempervirens 
+  code); or
+* when you have opted-out of sending other non-personal information, we record that an 
```

Comment:
We should state that information gathering of any kind is an opt-in mechanism. 

Also, do we gather the IP address in the case of a non opt-in event?